blog/use-google-optimize-in-react-web-for-a-b-testing.md

```diff
@@ -0,0 +1,136 @@
+# Use Google Optimize in React web for A/B Testing
+
+I'll explain how to create an experiment in the Google Optimize (GO) panel to get a simple boolean flag in our JavaScript code, specifically using Reactjs, to display the original version of our web, or the experiment.
+
+I won't focus on how to create experiments or specific configurations. I'll center this post on how to connect GO with JavaScript code.
+
+## Introduction
+
+I needed to integrate Google Optimize to do an [A/B testing](https://en.wikipedia.org/wiki/A/B_testing) in a project I'm working on. I found that GO was made for end users so that they can make little changes (text, CSS styles, etc.) in the website without needing a developer.
+
+But I couldn't find a good guide about how to receive a `bool` flag in the frontend to develop a basic `if` in JavaScript.
+
+## The Optimize panel
+
+### Create an experiment
+
+First, we should create an A/B test experiment using the Google Optimize panel. Make a click to the `Create experiment` button.
+
+![The create experience button](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/cqz1sgaks59l9tg4g9xl.png)
+
+Give a name to the experiment and select the `A/B test` option.
+
+![Select the experiment type](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/mrdrk8y301q6yt323fsp.png)
+
+When you link the experiment to a Google Analytics (GA) account the experiment receives an `Experiment ID`.
+
+![The experiment ID](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/uy5dttr74o384o7fy2ve.png)
+
+🧠 The developer will use this `ID` to read the flag in the JavaScript code. The experiment name is not used in the JavaScript side.
+
+Make sure you assign an objective to the experiment.
+
+![The objectives of the given experiment](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/2r477svv7i9f24c16yy9.png)
+
+Now, you can `start` the experiment.
+
+### Assign an activation event
+
+Review the `Activation event` setting, because you can't use the experiment as a flag in the JavaScript code until that event is fired.
```

Comment:
Do we have a synonym for 'experiment'? I think its used several times (more than it could be)